lib/hawkular/metrics/metric_api.rb

```diff
@@ -129,6 +129,19 @@ def get_data(id, starts: nil, ends: nil, bucketDuration: nil, buckets: nil, perc
         resp.is_a?(Array) ? resp : [] # API returns no content (empty Hash) instead of empty array
       end
 
+      # Retrieve raw data for multiple metrics.
+      # This API is experimental and may undergo non-backwards compatible changes on the future
```

Comment:
I'm just passing the warning that appears on the hawkular-metrics[1]. Should I remove?

[1] https://github.com/jsanda/hawkular-metrics/blob/e559e28482f51dd30805b63da09efce71538490b/api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/handler/CounterHandler.java#L268-L269